lib/aida/bot.ex

```diff
@@ -278,6 +278,8 @@ defmodule Aida.Bot do
   end
 
   def send_message(%{session: session} = message) do
+    session |> Session.save()
```

Comment:
Wasn't this needed before?

Reply:
It was, but was only on the survey side, it was missing on the scheduled messages side.